sdk/metricsadvisor/ai-metrics-advisor/recordings/browsers/api_key_metricsadvisorclient_feedback/recording_creates_comment_feedback.json

```diff
@@ -2,44 +2,44 @@
  "recordings": [
   {
    "method": "POST",
-   "url": "https://endpoint/metricsadvisor/v1.0/feedback/metric",
+   "url": "https://endpoint//metricsadvisor/v1.0/feedback/metric",
```

Comment:
why is the re-recording adding an extra `/` everywhere? I think this is why the tests are failing. @harsha-nalluru @jeremymeng

Reply:
Depends on what you gave as the replacement?

Reply:
ok I removed the trailing / in the replacement variable url